docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +29,76 @@ hosts_additional_entries:
 
 ## TLS certificates
 
-To enable external TLS encryption:
+:::warning
+To avoid unnecessary additional work and problems, it is recommended that you configure TLS with the intended target
+configuration of the specific environment during the initial rollout process.
```

Comment:
This is very vague, without the knowledge what can go wrong I don't think any user with deep knowledge about this topic can understand this warning.

What work? What problems? Imho a warning should be very specific what can go wrong why.